distributed/scheduler.py

```diff
@@ -480,7 +487,7 @@ def __init__(
         self.versions = versions or {}
         self.nanny = nanny
         self.status = status
-        self._hash = hash((address, pid, name))
+        self._hash = hash((address, pid, name, self._instance_id))
```

Comment:
Is there a point in having the other values mixed in?

Reply:
no, not really.